src/TrendsOverview.svelte

```diff
@@ -35,17 +35,20 @@
     setSelectedCounty,
     setSelectedState,
   } from "./choropleth.js";
+  import { fetchZipTrendLines, getZipsData, updateWithZipsData } from "./zip_files"
```

Comment:
fetchZipTrendLines and getZipsData appear to be unused.

Reply:
correct. It's used internally by updateWithZipsData. Removed from the import